lib/pghero/methods/connections.rb

```diff
@@ -5,6 +5,26 @@ def total_connections
         select_one("SELECT COUNT(*) FROM pg_stat_activity")
       end
 
+      def citus_worker_total_connections(nodesno)
```

Comment:
I think instead of passing `nodesno`, we can just get it here ourselves. (Same for the other function)